src/NodeDev.Blazor/Services/ProjectService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NodeDev.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NodeDev.Blazor.Services
+{
+    /// <summary>
+    /// Service used to keep a singleton of the project throughout the application.
+    /// </summary>
+    public class ProjectService
+    {
+        private Project _project;
+
+        public Project Project
+        {
+            get => _project;
+        }
```

Comment:
I didn't use "_" in front of private fields anywhere, we should keep it consistant
```suggestion
        public Project Project { get; private set; }
```